hub-templates/daskhub/Chart.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v2
+appVersion: '1.0'
+description: Deployment Chart for an ephemeral JupyterHub
+name: ephemeral-hub
```

```suggestion
description: Deployment Chart for a dask-enabled JupyterHub
name: daskhub
```
I think this new daskhub template is being referred to as `daskhub` in `hubs.yaml`